inc/commondbtm.class.php

```diff
@@ -167,13 +167,20 @@ class CommonDBTM extends CommonGLPI {
     */
    static $undisclosedFields = [];
 
+   /**
+    * Cache used to keep track of the last clone index
+    * Usefull when cloning an item multiple time to avoid iterating on the sql
+    * results multiple time while looking for an available unique name
+    * @var null|int
+    */
+   public static $last_clone_index = null;
```

Comment:
You are using this field on a non static context. I think it is preferable to not define it as static.
```suggestion
   public $last_clone_index = null;
```

Reply:
Done